src/init/features/project.ts

```diff
@@ -32,48 +34,47 @@ export interface ProjectInfo {
  */
 export async function getProjectInfo(options: any): Promise<ProjectInfo> {
```

Comment:
@mbleigh, @bkendall. I think I would like to reuse this method (in `apps:sdkconfig` command (https://github.com/firebase/firebase-tools/pull/1515)) => moving this to `management/projects.ts`

`projects.ts` is getting a bit long now. Do you want to split the methods into `projects-api.ts` and `projects-utils.ts`? 1 for api methods and 1 for logging and interactive flows in the commands?

Reply:
At the very least, I'd say let's not worry about `projects.ts` getting too long in this PR. Implement things with the current file structure, and we can look at refactoring in a separate PR if needed.